### PR TITLE
ALBS-75: Make CI on GH for the mirror service

### PR DESCRIPTION
**Pull request by soksanichenko**
- CI of checking the mirrors configs and the main config

### CODE REVIEW

**Review by rpisarev — COMMENTED** [review]
lgtm

**Review by rpisarev — APPROVED** [review]

**Review by anfimovdm — APPROVED** [review]

File: ci/config_checker.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        jsonschema.validate(
+            instance=config_data,
+            schema=json_schema,
+        )
+        return True, None
+    except jsonschema.ValidationError as err:
+        return False, err.message
+
+
+def main(args):
```
Comment (rpisarev) [inline ci/config_checker.py:103]: The good taste is using args and kwargs  parameters in every main() function

**Review by rpisarev — COMMENTED** [review]
lgtm

File: .github/workflows/json_schemas/service_config.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/Welcome4",
+  "definitions": {
+    "Welcome4": {
+      "type": "object",
+      "additionalProperties": False,
```
Comment (anfimovdm) [inline .github/workflows/json_schemas/service_config.json:7]: It's should be `false` in json

Comment (rpisarev) [inline .github/workflows/json_schemas/service_config.json:7]: lowcase?

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: mirrors.d/almalinux.repo.slimhost.com.ua.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: almalinux.repo.slimhost.com.ua
+address:
+  http: https://almalinux.repo.slimhost.com.ua/
```
Comment (rpisarev) [inline mirrors.d/almalinux.repo.slimhost.com.ua.yml:3]: it seems Ukraine, not Germany, hasn't got the idea of this

Comment (soksanichenko) [reply mirrors.d/almalinux.repo.slimhost.com.ua.yml:3]: It's not mine change, just from another PR. I can't to change anything where.

File: .github/workflows/json_schemas/service_config.json (hunk 2 above)
Comment (soksanichenko) [reply .github/workflows/json_schemas/service_config.json]: The file is removed as not needed

Comment (soksanichenko) [reply .github/workflows/json_schemas/service_config.json]: The file is removed as not needed

**Review by anfimovdm — APPROVED** [review]

**Comment by soksanichenko** [comment]
Closed as incorrect PR

**Pull request closed without merging**